src/test/common/fdc3.app-channels.ts

```diff
@@ -128,7 +128,7 @@ export function createAppChannelTests(cc: ChannelControl<any,any>, documentation
 
         const testChannel = await cc.createTestChannel("a-different-test-channel");
         await cc.setupAndValidateListener1(testChannel, "unexpected-context", errorMessage, () => { /*noop*/ })
```

Comment:
Based on suggested extra util function
```suggestion
        await cc.validateContextIsNotReceived(testChannel, null, errorMessage, () => { /*noop*/ })
```

Reply:
done in Kalina's PR